Assets/Xbox Live/Scripts/Social.cs

```diff
@@ -39,21 +40,41 @@ private void Awake()
         });
     }
 
+    private void Start()
+    {
+        if (this.XboxLiveUser == null)
+        {
+            this.XboxLiveUser = XboxLiveUserManager.Instance.GetSingleModeUser();
+        }
+    }
+
+    private void Update()
+    {
+        if (this.XboxLiveUser == null)
+        {
+            this.XboxLiveUser = XboxLiveUserManager.Instance.GetSingleModeUser();
+        }
+
```

Comment:
I'm actually testing something that might make this unneeded. Still testing it though.

---
In reply to: [120235112](https://github.com/Microsoft/xbox-live-unity-plugin/pull/66#discussion_r120235112) [](ancestors = 120235112)